docs/integrations/sigrid-api-documentation.md

```diff
@@ -584,12 +575,65 @@ DELETE https://sigrid-says.com/rest/analysis-results/api/v1/objectives/{customer
 This would delete the system-level test code ratio objective, if it exists at the system level, and 
 return 204. In case the objective does not exist at the system level, the endpoint returns 404. 
 
+#### Retrieving objectives status
+
+    GET https://sigrid-says.com/rest/analysis-results/api/v1/objectives-evaluation/{customer}?startDate={start}&endDate={end}
+    
+This will return the current objectives status and trend for all systems in your portfolio, for the specified time period. The `startDate` and `endDate` parameters accept dates in [ISO 8601 format](https://en.wikipedia.org/wiki/ISO_8601), for example 2024-05-17. 
+
+This end point returns the following response:
+
+<details markdown="1">
+  <summary>Example objectives status response</summary>
+
+```json
+{
+  "systems": [
+    "systemName": "my-example-system",
+      "objectives": [
+        {
+          "type": "OSH_MAX_LICENSE_RISK",
+          "feature": "OPEN_SOURCE_HEALTH",
+          "target": "LOW",
+          "targetMetAtStart": "NOT_MET",
+          "targetMetAtEnd": "NOT_MET",
+          "delta": "SIMILAR",
+          "stateAtEnd": "MEDIUM",
+          "level": "PORTFOLIO",
+          "parentId": 16
+        },
+        {
+          "type": "ARCHITECTURE_QUALITY",
+          "feature": "ARCHITECTURE_QUALITY",
+          "target": 4.0,
+          "targetMetAtStart": "NOT_MET",
+          "targetMetAtEnd": "NOT_MET",
+          "delta": "DETERIORATING",
+          "stateAtEnd": 3.68909,
+          "level": "SYSTEM"
+        }
+      ]
+    ]
+  }
+}
+```
+
+</details>
+
+The response contains every system in your portfolio, with each system containing all objectives that apply to that system. The objectives follow the same precedence rules as in Sigrid, so only objectives that actually apply will be included in the results.
```

Comment:
Suggestion: try to avoid the term "apply". In principle, all portfolio objectives apply to all systems at all time, however, if the metadata of a system doesn't match the condition(s) of the objective, it doesn't result in a target.

This seems like hair splitting, but we already witnessed a case were a user thought that each portfolio objective, next to conditions, had a list of systems to which it applies (that user was asking where they could edit said list).

```suggestion
The response contains every system in your portfolio, with each system containing all objectives that apply to that system. The objectives follow the same precedence rules as in Sigrid, so only objectives with conditions that match the metadata of the system are included in the result.
```